src/ParseQuery.js

```diff
@@ -2111,6 +2122,26 @@ class ParseQuery {
       this._xhrRequest.onchange();
     };
   }
+
+  /**
+   * Sets a comment to the query so that the query
+   *can be identified when using a the profiler for MongoDB.
+   *
+   * @param {string} value a comment can make your profile data easier to interpret and trace.
+   * @returns {Parse.Query} Returns the query, so you can chain this call.
+   */
+  comment(value: string): ParseQuery {
+    if (typeof value === 'undefined') {
```

Comment:
```suggestion
    if (value == null) {
```

Simplify, so that undefined and null values are considered "no value"?